### PR TITLE
Add PayPal Business variant

### PR DESCRIPTION
**Pull request by service-paradis**
# Description

Link PayPal Business App to PayPal icon
https://play.google.com/store/apps/details?id=com.paypal.merchant.client&hl=en_US

### Icons linked
* PayPal Business (linked `com.paypal.merchant.client` to `@drawable/paypal`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

**Review by Chefski — COMMENTED** [review]
What if PayPal Business could have a small B in the bottom right corner similar to Chrome Dev and other icons which have different variations? This would help distinguish the normal PayPal from PayPal Business

**Comment by service-paradis** [comment]
You're right, would be a better idea to have some kind of differentiation between the apps! I'll see if I can propose an icon for this next week :)

**Comment by service-paradis** [comment]
I just made a proposal for a Business variant!

File: svgs/paypal_business.svg
Comment (Chefski) [inline svgs/paypal_business.svg:1]: The two P's have been changed to a shape. Could you leave them as stroke please? It'll make it easier to make changes in the future. Other than that the icon is good!

Comment (KTibow) [reply svgs/paypal_business.svg:1]: (They mean how it's saved as a fill right now)

**Comment by Chefski** [comment]
I suppose it can get merged since there's many more icons that need to be fixed. Thanks for checking

**Pull request merged**